gradlew

```diff
@@ -28,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
Any reason for these?

Reply:
`gradlew` is the wrapper code, which is auto-generated. We shouldn't change it

Reply:
This was just the output from `./gradlew wrapper` after updating the version in the `build.gradle`.